src/Components/Nav/Searchbar/ResultView/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class ResultView extends Component {
+  render() {
+    const { searchResult, searchbarOff } = this.props;
+    return (
+      <>
+        {searchResult.length === 0 ? (
+          <div className="noResult">검색결과가 없습니다.</div>
+        ) : (
+          <ul>
```

Comment:
```suggestion
  render() {
    const { searchResult, searchbarOff } = this.props;
    return (
      if (!searchResult.length)
        return <div className="noResult">검색결과가 없습니다.</div>
    
      return (<ul>
```
- 길이가 0이라는 것은 false라는 것과도 동일하기 때문에 `===` 대신 `!`로 표현하는 것도 가능합니다.
- early return 을 사용하면 fragment와 삼항 연산자를 줄일 수 있을 듯 합니다.

Reply:
수정하였슴니다!